src/sections/Careers/Careers-Programs-single/ProgramsSingle.style.js

```diff
@@ -1,7 +1,30 @@
 import styled from "styled-components";
 
 const ProgramsPageWrapper = styled.div`
- margin-bottom: 30px;
+margin: 4.5rem 0 3.125rem 0;
```

Comment:
The margin is probably going be messed up due to this. You need to also account for the previous `margin-bottom: 30px;`